internal/eval/fold.go

```diff
@@ -109,6 +109,32 @@ func fold(n ast.IsNode) ast.IsNode {
 				return ast.NodeTypeHas{StrOpNode: ast.StrOpNode{Arg: nodes[0], Value: v.Value}}
 			},
 		)
+	case ast.NodeTypeGetTag:
+		return tryFold(
+			[]ast.IsNode{v.Left, v.Right},
+			func(values []types.Value) Evaler {
+				if _, ok := values[0].(types.EntityUID); ok {
```

Comment:
You could probably just return an error always for both this and HasTag

```
func(values []types.Value) Evaler {
    return newErrorEval(fmt.Errorf("fold.GetTag.EntityUID"))
}
```

Reply:
Yes, done here: https://github.com/cedar-policy/cedar-go/pull/77/commits/36fd3e7573bac8103b345a0c8d413d6eef37dccb.